documents/openapi/openapi.yaml

```diff
@@ -2776,15 +2823,19 @@ components:
             uri:
               type: string
               format: uri
-        scheduled_for:
+        uri:
           type: string
+          format: uri
```

Comment:
I think the indentation is wrong.  The `uri` attribute is a property of the embedded `template` object on line 2769/2816.  Deleting line 2766 is correct because there is no top level `uri` attribute for a Notification response according to the `serialize` method of the Notification class in app/models.py.

Reply:
uri was there to begin with, I just moved it.

a uri is also present in the POST response, which is what this schema represents. it's for the GET route.
```
{
    "billing_code": null,
    "callback_url": null,
    "content": {
        "body": "# Cat Ipsum Generator Info\n\nVisit the [Cat Ipsum Text Generator](http://www.catipsum.com/index.php) for some chunky text.\n\n## Congrats!\n\n__QA__ thanks _you_.",
        "subject": "A message from QA"
    },
    "id": "2a79593d-34a0-4089-acd8-3cfdf3847935",
    "reference": null,
    "scheduled_for": null,
    "template": {
        "id": "52bfd6eb-dc89-4bee-a478-84d50f5af673",
        "uri": "https://dev-api.va.gov/services/c6340735-40fb-4ed8-ada1-1cbc34cdfaf0/templates/52bfd6eb-dc89-4bee-a478-84d50f5af673",
        "version": 1
    },
    "uri": "https://dev-api.va.gov/v2/notifications/2a79593d-34a0-4089-acd8-3cfdf3847935"
}
```

Reply:
We discussed this.  It's right the way it is.